### PR TITLE
close #8074, update mouse and input coordinates page

### LINKED ISSUE #8074: Out of date: Mouse and input coordinates page
**Your Godot version:**
4.1.1

**Issue description:**
The GDScript code snippet is not functional in 4.1.1, since get_viewport_rect() no longer exists.

Please update
`print("Viewport Resolution is: ", get_viewport_rect().size)`
to
`print("Viewport Resolution is: ", get_viewport().size)`

**URL to the documentation page:**
https://docs.godotengine.org/en/stable/tutorials/inputs/mouse_and_input_coordinates.html

If you know how to fix the issue you are reporting please
consider opening a pull request. We provide a tutorial on
using git here: https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html,
writing documentation at https://docs.godotengine.org/en/stable/community/contributing/docs_writing_guidelines.html
and contributing to the class reference here: https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html

### PR DESCRIPTION
**Pull request by Hex-4**
closes #8074

Tiny PR that updates the syntax on the mouse and input coordinates page.

### CODE REVIEW

File: tutorials/inputs/mouse_and_input_coordinates.rst [hunk 1]
```diff
@@ -36,7 +36,7 @@ for example:
            print("Mouse Motion at: ", event.position)
 
        # Print the size of the viewport.
-       print("Viewport Resolution is: ", get_viewport_rect().size)
+       print("Viewport Resolution is: ", get_viewport().size)
```
Comment (Sauermann) [inline tutorials/inputs/mouse_and_input_coordinates.rst:39]:
```suggestion
       print("Viewport Resolution is: ", get_viewport().get_visible_rect())
```

`get_viewport_rect()` is defined in `CanvasItem` nodes. So the previous description doesn't work for all nodes and should be updated.
However `get_viewport_rect()` doesn't return `get_viewport().size`, but `get_viewport().get_visible_rect()`, so it should be changed accordingly in order to keep the same behavior as the previous description.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Tiny PR but fixes people running into outdated, non-working code. Great! :) Merged. Thanks Sauermann for the review, and thanks Hex-4 for oncorporating it so quickly and squashing. Nice!

File: tutorials/inputs/mouse_and_input_coordinates.rst [hunk 2]
```diff
@@ -49,7 +49,7 @@ for example:
             GD.Print("Mouse Motion at: ", eventMouseMotion.Position);
 
         // Print the size of the viewport.
-        GD.Print("Viewport Resolution is: ", GetViewportRect().Size);
+        GD.Print("Viewport Resolution is: ", GetViewportRect().GetVisibleRect());
```
Comment (raulsntos) [inline tutorials/inputs/mouse_and_input_coordinates.rst:52]: The C# example doesn't match the GDScript example.

```suggestion
        GD.Print("Viewport Resolution is: ", GetViewport().GetVisibleRect());
```

**Comment by Hex-4** [comment]
Looks like there are a few changes I should make - do I need to make a new PR or can I just amend another commit to this one?

**Comment by Sauermann** [comment]
You can amend PRs only before they are merged. So creating a new PR is the only viable option.

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.1

**Pull request merged**